pages/dkp/kommander/2.1/install/networked/index.md

```diff
@@ -43,7 +43,7 @@ More information on setting a StorageClass as default can be found at [Changing
 
 Before running the commands below, ensure that your `kubectl` configuration references the cluster on which you want to install Kommander. You can do this by setting the `KUBECONFIG` environment variable to the appropriate kubeconfig file's location.
 
-To install Kommander with HTTP proxy setting enabled, you need to follow the instructions outlined in [enable Gatekeeper][enable-gatekeeper] section before proceeding further. To enable a Gatekeeper proxy, you must pass the `install.yaml` (or whatever the config file was named) you created to the following command using `--installer-config=install.yaml`
+<p class="message--note"><strong>NOTE:</strong> See the <a href="../configuration">configuration</a> page for details on how to customize a Kommander installation.</p>
```

Comment:
```suggestion
<p class="message--note"><strong>NOTE:</strong> See the <a href="../configuration">configuration</a> page for more details on how to customize a Kommander installation.</p>
```

Reply:
resolved in a2389daa9fd58b84770bb55c6ee73e7b4e829e54